hraven-etl/src/test/java/com/twitter/hraven/etl/TestJobHistoryFileParserHadoop2.java

```diff
@@ -90,7 +76,7 @@ public void testCreateJobHistoryFileParserCorrectCreation() throws IOException {
             JobHistoryKeys.JOB_STATUS.toString().toLowerCase()));
     
     assertNotNull(statusRecord);
-    assertEquals((String)statusRecord, JobHistoryFileParserHadoop2.JOB_STATUS_SUCCEEDED);
+    assertEquals((String)statusRecord, JobHistoryFileParserHadoop2.JOB_STATUS_SUCCESS);
```

Comment:
This is just a small name change from JOB_STATUS_SUCCEEDED to JOB_STATUS_SUCCESS no logic change ...